config/cars/mods/urd/urd_t5_aura_2018.ini

```diff
@@ -0,0 +1,97 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
```

Comment:
Emissive multiplier strength seems high, re-check with different filters to ensure visual consistency for rollout.